### PR TITLE
MC-10232 Updates user to reflect async changes

### PR DESCRIPTION
**Pull request by m-orfanos**
Updates user documentation to reflect async changes.

### CODE REVIEW

File: source/includes/administration/_users.md [hunk 1]
```diff
@@ -174,7 +209,13 @@ curl -X POST "https://cloudmc_endpoint/v2/users" \
 }
 ```
 
-Create a user in a specific organization. There's two different types of [role](#administration-roles) you can assign to the user. A system role will determine the set of system permissions the user will have. An environment role will give the user access to an environment and will determine what he can see and do in that environment. You will need the `Create a new user` permission to execute this operation.
+Create a user in a specific organization.
+
+Users are created asynchronously on the underlying service(s) and progress is reflected in the state of the user's service connection(s).
+
+There's two different types of [role](#administration-roles) you can assign to the user. A system role will determine the set of system permissions the user will have. An environment role will give the user access to an environment and will determine what he can see and do in that environment.
```
Comment (JamieWulala) [inline source/includes/administration/_users.md]: `there are two different types`

Comment (JamieWulala) [reply source/includes/administration/_users.md]: last sentence `what he can see` -> `what one can see`

Comment (JamieWulala) [reply source/includes/administration/_users.md]: instead of `see and do` maybe use `access rights` or something else?

**Review by JamieWulala — APPROVED** [review]
LGTM, minor changes

File: source/includes/administration/_users.md (hunk 1 above)
Comment (m-orfanos) [reply source/includes/administration/_users.md]: I rewrote the sentence a bit. 493047c4621e951d804d9017c3f60d0b543f06d4

File: source/includes/administration/_users.md [hunk 2]
```diff
@@ -52,14 +69,15 @@ Retrieve information about users you have access to. If you want access to other
...
+`creationDate`<br/>*string* | The date in [ISO 8601](https://en.wikipedia.org/wiki/ISO_8601) that the user was created
+`email`<br/>*string* | The email of the user
 `firstName`<br/>*string* | The first name of the user
 `lastName`<br/>*string* | The last name of the user
-`email`<br/>*string* | The email of the user
-`creationDate`<br/>*string* | The date in [ISO 8601](https://en.wikipedia.org/wiki/ISO_8601) that the user was created
-`status`<br/>*string* | The current status of the user.
 `organization`<br/>*[Organization](#administration-organization)* | The organization to which the user belongs
 `roles`<br/>*Array[[Role](#administration-roles)]* | The system and environments roles that are assigned to the user<br/>*includes*: `id`, `name` and `environment.id`
+`serviceConnection`<br/>*Array[[ServiceConnection](#administration-service-connections)]* | The service connection of the user<br/>*includes*: `id`, `name`, `serviceCode`, `type`, and `state`
```
Comment (Shabirmean) [inline source/includes/administration/_users.md]: serviceConnection**(s)**?

Comment (m-orfanos) [reply source/includes/administration/_users.md]: nice catch :p

File: source/includes/administration/_users.md [hunk 3]
```diff
@@ -123,21 +158,20 @@ Retrieve information about a specific user. If you want access to other users in
...
-`email`<br/>*string* | The email of the user
-`creationDate`<br/>*string* | The date in [ISO 8601](https://en.wikipedia.org/wiki/ISO_8601) that the user was created
-`status`<br/>*string* | The current status of the user.
-`primaryRoleBinding`<br/>*RoleBinding* | The primary role assigned to this user. This role will always be a fixed role.
 `organization`<br/>*[Organization](#administration-organization)* | The organization to which the user belongs
-`environments`<br/>*Array[[Environment](#administration-environments)]* | The environments the user is member of<br/>*includes*: `id`, `name`
+`primaryRoleBinding`<br/>*RoleBinding* | The primary role assigned to this user. This role will always be a fixed role.
 `roles`<br/>*Array[[Role](#administration-roles)]* | The system and environments roles that are assigned to the user<br/>*includes*: `id`, `name` and `environment.id`
-
+`serviceConnection`<br/>*Array[[ServiceConnection](#administration-service-connections)]* | The service connection of the user<br/>*includes*: `id`, `name`, `serviceCode`, `type`, and `state`
```
Comment (Shabirmean) [inline source/includes/administration/_users.md]: Same as above

File: source/includes/administration/_users.md [hunk 4]
```diff
@@ -123,21 +158,20 @@ Retrieve information about a specific user. If you want access to other users in
...
+`status`<br/>*string* | The current status of the user.
+`userName`<br/>*string* | The username of the user
```
Comment (Shabirmean) [inline source/includes/administration/_users.md:171]: Is there any particular ordering of the fields? Should we put things like the this on top and Json objects, arrays after?

File: source/includes/administration/_users.md [hunk 5]
```diff
@@ -123,21 +158,20 @@ Retrieve information about a specific user. If you want access to other users in
...
-`creationDate`<br/>*string* | The date in [ISO 8601](https://en.wikipedia.org/wiki/ISO_8601) that the user was created
-`status`<br/>*string* | The current status of the user.
-`primaryRoleBinding`<br/>*RoleBinding* | The primary role assigned to this user. This role will always be a fixed role.
 `organization`<br/>*[Organization](#administration-organization)* | The organization to which the user belongs
-`environments`<br/>*Array[[Environment](#administration-environments)]* | The environments the user is member of<br/>*includes*: `id`, `name`
+`primaryRoleBinding`<br/>*RoleBinding* | The primary role assigned to this user. This role will always be a fixed role.
 `roles`<br/>*Array[[Role](#administration-roles)]* | The system and environments roles that are assigned to the user<br/>*includes*: `id`, `name` and `environment.id`
-
+`serviceConnection`<br/>*Array[[ServiceConnection](#administration-service-connections)]* | The service connection of the user<br/>*includes*: `id`, `name`, `serviceCode`, `type`, and `state`
+`status`<br/>*string* | The current status of the user.
```
Comment (Shabirmean) [inline source/includes/administration/_users.md:170]: Should we be more elaborate on what the status means? Im not sure. Just asking.

**Review by Shabirmean — APPROVED** [review]
Minor suggestions. LGTM :shipit:

File: source/includes/administration/_users.md (hunk 5 above)
Comment (m-orfanos) [reply source/includes/administration/_users.md:170]: The status here is ACTIVE/INACTIVE. I don't know if there are other states. I can say something like "Can either be ACTIVE or INACTIVE".

Comment (Shabirmean) [reply source/includes/administration/_users.md:170]: Cool. I was thinking it was related to LRO

File: source/includes/administration/_users.md (hunk 4 above)
Comment (m-orfanos) [reply source/includes/administration/_users.md:171]: I sorted them a-z. IMO it's easier to find variables like that. Is there something the team uses by default?

Comment (Shabirmean) [reply source/includes/administration/_users.md:171]: I am not exactly sure if there is a default. I usually arrange it in the order they appear in the response object. But sorted order makes sense too.

File: source/includes/administration/_users.md (hunk 3 above)
Comment (m-orfanos) [reply source/includes/administration/_users.md]: 63c7859

File: source/includes/administration/_users.md (hunk 2 above)
Comment (m-orfanos) [reply source/includes/administration/_users.md]: 63c7859

**Pull request merged**